backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationResolver.java

```diff
@@ -14,12 +19,35 @@
 public class OrganizationResolver implements GraphQLQueryResolver  {
 
     private ApiUserService _userService;
+    private DeviceTypeService _deviceService;
 
-    public OrganizationResolver(OrganizationService os, ApiUserService users) {
+    public OrganizationResolver(OrganizationService os, ApiUserService users, DeviceTypeService devices) {
         _userService = users;
+        _deviceService = devices;
     }
 
     public Organization getOrganization() {
+
+        List<DeviceType> currentDevices = _deviceService.fetchDeviceTypes();
+        ArrayList<String> currentDeviceNames = new ArrayList<String>();
```

Comment:
This hurts me to read, just a little. It's fine, though, really. 😁 

```suggestion
        List<String> currentDeviceNames = currentDevices.stream().map(d->d.getName()).collect(Collectors.toList());
```

Reply:
lol look at all that fancy function chaining and java-ing!